<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <widget class="QProgressBar" name="pBar_main">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="cb_progress_value"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb_progress">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Направление</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rb_up">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_down">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
